LetSwift/ApplicationViewControllers/Photos/SinglePhotoViewController.swift

```diff
@@ -89,7 +89,7 @@ final class SinglePhotoViewController: UIViewController {
         spinnerView.translatesAutoresizingMaskIntoConstraints = false
         spinnerView.thicknessRatio = 0.25
         spinnerView.trackTintColor = .paleGrey
-        spinnerView.progressTintColor = .swiftOrange
+        spinnerView.progressTintColor = EventBranding.current.color
```

Comment:
instead of explicitly calling branding event you could hide it in uicolor extension:

```
extension UIColor {
    
    class var brandColor: UIColor {
        return EventBranding.current.color
    }
}

```

thus we have less dependency on this object/class